### ISSUE TITLE
higher val_loss have higher val_loss, why?

### ISSUE THREAD

**Issue by Jhy1993** [fchollet/keras#3559]
with the training:
        Q1: train_acc keep increase but the val_acc oscillation
        Q2: higher val_loss have higher val_loss

**Comment by carlthome** [comment]
**Q1:** 
Obviously training accuracy will keep increasing ([see overfitting](https://en.wikipedia.org/wiki/Overfitting)) as that's the data the model uses for weight updates.

Validation accuracy will typically oscillate heavily if the validation data is too small and dissimilar to the training data, but you should always expect it to oscillate. Average the validation accuracy curve and look for positive trends instead.

**Q2:** 
Higher val_loss have higher val_loss? Care to elaborate a bit?

**Comment by Jhy1993** [comment]
@carlthome  
Thanks for your answer!
about Q2:
usually lower val_loss have higher accuracy,

but i meet higher val_loss have higher accuracy.
like this:
epoch 55          val_loss:1.3  accuracy: 85%
epoch 56         val_loss:1.4  accuracy :86%

**Comment by alex-j-j** [comment]
Smaller loss does not necessarily mean higher accuracy, it can mean that the right predictions were made with higher confidence. This also means that the accuracy can actually increase a bit if the loss increases as well.

**Comment by Jhy1993** [comment]
@alex-j-j 
Thank you very much!
